plugins/modules/remote_management/hpilo/hpilo_info.py

```diff
@@ -122,6 +122,226 @@
     type: str
     sample: ON
     version_added: 3.5.0
+
+server_name:
+    description: Server name
+    returned: always
+    type: str
+    sample: milo.company.com
```

Comment:
```suggestion
    sample: milo.company.com
    version_added: 4.1.0
```
Same for the others.